src/gdcdictionary/schemas/follow_up.yaml

```diff
@@ -2953,11 +2988,26 @@ properties:
       - Post Secondary Therapy
       - Postoperative
       - Preoperative
+      - Prior to Adjuvant Therapy
       - Prior to Diagnosis
       - Prior to Treatment
       - Recurrence/Progression
+      - Within 2 Months After Completion of First-Course Treatment
       - Within 3 Months of Surgery
       - Other
+      - Unknown
+      - Not Reported
+
+  treatment_emergent_adverse_event:
+    $ref:
+      - "_terms.yaml#/treatment_emergent_adverse_event/common"
+    enum:
+      - Definitely
+      - Possibly
+      - Probably
+      - Unlikely
+      - Unrelated
+      - Unknown
       - Not Reported
```

Comment:
confused about why this Not Reported was already here. I think it was already there previously for timepoint category and git is just confused?
^^ if so, should we remove "Not Reported" from timepoint_category in the jira?

Reply:
Yeah I saw that too, I think its just the way that git rendered the diff for changes. I think the timepoint_category enum for Not Reported should remain so we are consistent that all enum type props have `Unknown` and `Not Reported`

Reply:
oh, i mean if timepoint_category already had "Not Reported", then we don't need to claim we added it for Leeuvenhoek in the jira ticket. But not wedded to this in any way.